### ISSUE TITLE
No recipes in NEI with Forestry 3.6.3

### ISSUE THREAD

**Issue by JoeTheFkingFrypan** [Tonius/NEI-Integration#82]
Hi,

I'd like to report that the latest version of `NEI-Addons` (__v1.0.11__) does not provide recipes for Forestry machines or combs outputs in NEI. (_You can see by whay bee species a comb is produced, but not what it will provide when thrown in a centrifuge_).

Versions of mods used :

    `NEI-Addons` v.1.12.11.36
    `NEI-Integration` v1.0.11
    `Forestry` v3.6.3

Best regards.

**Comment by Tonius** [comment]
I can't seem to reproduce this issue; recipes in the Centrifuge and all other Forestry machines work just fine for me.

Which combs' products exactly are you trying to look up?

**Comment by VaporStrike** [comment]
I have the same issue. Can see bee breeding results but not the recipes for the machines or the machine output recipes.

NEI addons version 1.12.8.29
NEI Integration version 1.0.9
Forestry version 3.5.6.15

Yes, I know everything is out of date, but we're running an older version of Direwolf. I am going to try to update both addons and integration if possible, and return here afterwards.

**Comment by GAGNAIRE-Romain** [comment]
I just double-checked but I couldn't get any comb to work.

The only usage displayed in NEI for combs is "calcination" that comes from Equivalent Exchange 3.

Could it be related to NEI itself ? 
I'm currently using NEI `v.1.0.5.111`

Maybe downgrading NEI would help.

**Comment by Tonius** [comment]
Nope, still working fine for me with those versions of NEI, Addons and Forestry.

Some mod you have installed may be messing with NEI or even the Centrifuge recipes themselves. Try removing mods or adding them from scratch, one by one, and see if it starts working.

**Comment by VaporStrike** [comment]
What was the issue that caused this in Direwolf 1.4.1? He could be having a similar issue, and that might help him figure out what was causing it.

**Comment by belathus** [comment]
So, a followup to this, as I've been seeing something similar with Railcraft recipes as well as Forestry's recipes...

NEI does not show any of these recipes if I start Minecraft, then immediately hop on a multiplayer server. But if I join a singleplayer world first, the recipes show up just fine, even if I then subsequently join a multiplayer server.
